app/src/main/java/com/globant/counter/mvp/view/CountView.kt

```diff
@@ -2,25 +2,68 @@ package com.globant.counter.mvp.view
 
 
 import android.app.Activity
+import android.text.TextUtils
+import com.globant.counter.mvp.model.CalculatorItem
 import com.globant.counter.utils.bus.RxBus
-import com.globant.counter.utils.bus.observer.OnCountButtonPressedBusObserver
-import com.globant.counter.utils.bus.observer.OnResetButtonPressedBusObserver
+import com.globant.counter.utils.bus.observer.OnCalculatorActionItemPressedBusObserver
+import com.globant.counter.utils.bus.observer.OnCalculatorClearButtonPressedBusObserver
+import com.globant.counter.utils.bus.observer.OnCalculatorEqualButtonPressedBusObserver
+import com.globant.counter.utils.bus.observer.OnCalculatorNumberButtonPressedBusObserver
 import kotlinx.android.synthetic.main.activity_main.*
 
 
 class CountView(activity: Activity) : ActivityView(activity) {
+    fun setExpression(expression: String) {
+        activity?.textExpression?.text = expression
+    }
 
     init {
-        activity.count_button.setOnClickListener {
-            RxBus.post(OnCountButtonPressedBusObserver.OnCountButtonPressed())
+        activity.btnSum.setOnClickListener {
```

Comment:
you could create a function that does this same logic receiving `CalculatorItem`
```suggestion
        activity.btnSum.setOnClickListener {
            postOperation(CalculatorItem.ADDITION)
        }
        ...
fun postOperation(operation: CalculatorItem) {
    getInput()
    RxBus.post(OnCalculatorActionItemPressedBusObserver.OnCalculatorActionButtonPressed(operation))
}
```